lib/beaker/options/parser.rb

```diff
@@ -216,8 +217,13 @@ def parse_args(args = ARGV)
         cmd_line_options[:command_line] = ([$0] + args).join(' ')
         @attribution = @attribution.merge(tag_sources(cmd_line_options, "flag"))
 
-        # Subcommands are the first to get merged into presets
-        subcommand_options = Beaker::Options::SubcommandOptionsParser.parse_subcommand_options(args)
+        # Global subcommand options from $HOME/.beaker/subcommand_options.yaml are first to get merged into presets
+        homedir_options = Beaker::Options::SubcommandOptionsParser.parse_subcommand_options(args, ENV['HOME']+'/.beaker/subcommand_options.yaml')
```

Comment:
the string here is the same as `SUBCOMMAND_OPTIONS` used below, [right](https://github.com/puppetlabs/beaker/blob/master/lib/beaker/subcommands/subcommand_util.rb#L15)? If that's the case, why not use that same thing here?

Also, these lines are getting crazy long. Maybe set the file names on separate lines? Or, it seems like this code in this block & the one below is the same code, is it not? If that's the case, why not loop over the file sources and not have the duplicate code?